### PR TITLE
CSS Frameworks CA - Woopsie, a social media application

### PR DESCRIPTION
**Pull request by Berrinj**
This PR provides changes that makes the application work on all screen sizes (mobile>desktop). 

Features:
- [x] Login page with HTML validation
- [x] Profile page with a list of user posts, profile info and a follow button.
- [x] Feed page with posts, search bar, sort option and form to create posts.

Uses:
- SASS
- LiveServer
- Bootstrap

### CODE REVIEW

File: index.html
Comment (ttowntom) [inline index.html:1]: There is no container for the content, which results in the page stretching to whatever width.

File: profile/index.html
Comment (ttowntom) [inline profile/index.html:1]: There is no container for the content, which results in the page stretching to whatever width.

File: feed/index.html
Comment (ttowntom) [inline feed/index.html:1]: There is no container for the content, which results in the page stretching to whatever width.

File: npm
Comment (ttowntom) [inline npm:1]: Should this file be here at all?

File: package.json
Comment (ttowntom) [inline package.json:1]: You could add more useful and correct info, like your name and email under author, the correct url for repo and so on.

File: npm
Comment (Berrinj) [reply npm:1]: deleted.

File: index.html
Comment (Berrinj) [reply index.html:1]: added max width on footer, but the header is still unsolved.

File: feed/index.html
Comment (Berrinj) [reply feed/index.html:1]: added max width on footer, but the header is still unsolved.

File: profile/index.html
Comment (Berrinj) [reply profile/index.html:1]: added max width on footer, but the header is still unsolved.

File: src/scss/_customs.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  5: $spacer * 3,
+  6: $spacer * 6,
+  7: $spacer * 9,
+);
+
+//Links
+$link-decoration: none;
+$link-hover-decoration: italic;
+
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
```
Comment (BergitTveit) [inline src/scss/_customs.scss:19]: To follow the convention of clean code, please add an empty line at the bottom of the file.  

- This visually separates the document's end from the file's end, aiding navigation and comprehension.

- Compatibility: Some older systems may struggle to parse files lacking a newline character at the end, causing potential issues.

- Concatenation: Ensures proper separation of concatenated files, preventing unexpected behavior in the output.

Please apply to all files.

Comment (Berrinj) [reply src/scss/_customs.scss]: applied!

**Pull request merged**